README.md

```diff
@@ -75,6 +75,35 @@ For example, to compile and run the server example, use:
 make examples/server
 ```
 
+### CMake Build
+
+```bash
+# default config (tests and examples build OFF, static library)
+cmake -B path/to/build_directory
```

Comment:
I suggest including the build type here — e.g., `cmake -DCMAKE_BUILD_TYPE=Debug -S . -B bld/debug` and/or `cmake -DCMAKE_BUILD_TYPE=Release -S . -B bld/release`. (I use both, and switch between them depending on what I'm currently doing.)

Reply:
If no choose `CMAKE_BUILD_TYPE`, debug is default.